frontend/templates/product/sections/FeaturedProductsSection/index.tsx

```diff
@@ -108,13 +116,43 @@ function ProductGridItem({
    isPro,
    warranty,
    oemPartnership,
+   sku,
 }: ProductGridItemProps) {
    const discountPercentage = computeDiscountPercentage(price, compareAtPrice);
    const hasLifetimeWarranty =
       typeof warranty === 'string' && isLifetimeWarranty(warranty);
 
    const productHeadingId = `product-heading-${handle}`;
 
+   const appContext = useAppContext();
+   const fetchData = async () => {
+      if (!sku) {
+         return null;
+      }
+      const reviewsData = await fetchProductReviews(
+         appContext.ifixitOrigin,
+         sku
+      );
+      return reviewsData;
+   };
+
+   const [reviewsData, initReviews] = useState<
+      ProductReviewData | null | undefined
+   >();
+   useEffect(() => {
+      fetchData()
```

Comment:
This is a little briefer and obviates the need for the `fetchData` callback

```suggestion
      if (!sku) {
         return;
      }
      fetchProductReviews(appContext.ifixitOrigin, sku)
```